docs/src/gs/customize-worker-node.md

```diff
@@ -0,0 +1,77 @@
+# Customizing worker nodes
+## Configure user managed boot volume encryption
+Use the following configuration in OCIMachineTemplate to use a customer
+managed boot volume encryption key
```

Comment:
should we link out to OCI docs that talk about encryption keys? I think if someone comes here looking how to setup encryption keys they probably know how to do it, but on the chance that a user is just looking around it would be nice for them to figure out from our docs how to setup those keys.

Reply:
fixed